### ISSUE TITLE
[ENH]: UserWarning: The figure layout has changed to tight

### ISSUE THREAD

**Issue by barnakissdr** [matplotlib/matplotlib#26535]
### Problem

plt.tight_layout() gives UserWarning: The figure layout has changed to tight.

That's what tight_layout should do, right? Then why to annoy us with a warning saying that tight_layout did what it was supposed to do?

### Proposed solution

Remove UserWarning

**Comment by rcomer** [comment]
Can you say more about the circumstances where you see the warning?  If you set up your figure with constrained layout and then called `tight_layout`, then these two things are mutually exclusive so we throw the warning because we think trying to use both was a mistake.  If you are seeing the warning because you called `tight_layout` twice then that was a mistake in Matplotlib and has been fixed ready for the next release (#26300).

**Comment by barnakissdr** [comment]
You’re right: I called tight_layout twice. Now that I removed the first one the warning has disappeared. Thanks!

 

Barna

**Comment by rcomer** [comment]
OK great, are you happy to close this issue then?  With version 3.8, you should be able to call `tight_layout` as many times as you want and the warning should not appear.  The release candidate is out on PyPI if you want to try it out.
https://pypi.org/project/matplotlib/3.8.0rc1/

**Comment by barnakissdr** [comment]
Sure; I closed it. Thanks again!

 

Barna
